### DISCUSSION TITLE
Something went wrong in payment sheet

### DISCUSSION THREAD

**Discussion by iospidigi** [flutter-stripe/flutter_stripe#1653] [Q&A]
**Describe the bug**
While making payment in Stripe, the payment sheet shows "Something went wrong" message after entering the card details.

**To Reproduce**
1. Entering number 123-456-789 into the cardfield.
2. Enter expiry and cvv.
3. Tapping the confirm button.

**Expected behavior**
Payment should be successful 

**Smartphone / tablet**
 - Device: Vivo v27
 - OS: Funtouch OS Android 14
 - Package version: 9.3.0
 - Flutter version: 3.10.6

**Comment by Bittibw** [comment]
Go on the stripe payment gateway website and go to the payment section and click any of the payment and go below to read the logs why the payment is being cancel and share it here.

**Reply by RaviVaniyaTatva** [reply]
Do we need to add our real credit card in Stripe for the test environment? because I am getting something went wrong on payment sheet and on the Stripe dashboard, it says Payment Incomplete with this error

- An attempt to fulfil the payment pi_3P14iGSB43qsSKcF0w29eJNm for £100.00 failed
- The payment pi_3P14iGSB43qsSKcF0w29eJNm for £100.00 requires you to take action in order to complete the payment
